test/src/api/responderTest.php

```diff
@@ -7,6 +7,33 @@ function auth_session_exists() {
     return $dummyUserLoggedIn;
 }
 
+// Dieroller values which may change over time
+
+// RandomBM skills (used by all RandomBM types except RandomBMFixed)
+$RANDOMBM_SKILL = array(
```

Comment:
One thought here:

What would you think about doing things a little differently?

    $RANDOMBM_SKILL_ARRAY = array(
        'B', // Berserk
        'b', // Boom
        'c', // Chance
        ..., 
        'h' // Weak
    );
    $RANDOMBM_SKILL = array_flip($RANDOMBM_SKILL_ARRAY);

The reason I'm suggesting this is that any time a new skill is added, there is literally one change, instead of all the numbers after the change needing to be updated.